### PR TITLE
CASSANDRA-17696 WEBSITE - Updating Ecosystem information for Anant

### PR DESCRIPTION
**Pull request by adp8ke**
Updating Ecoystem webpage for Anant

### CODE REVIEW

File: site-content/source/modules/ROOT/pages/ecosystem.adoc [hunk 1]
```diff
@@ -69,7 +69,9 @@ https://ignite.apache.org/[Apache Ignite,window=blank]: Use Ignite as a traditio
 
 https://github.com/Azure-Samples/cassandra-proxy[Azure-Samples/Cassandra Proxy,window=blank]: This proxy handles client connections and forwards them to two Cassandra clusters simultaneously.
 
-http://cassandra.link/[Cassandra.link,window=blank]: Curated site with tools, along with cassandra.tools.
+http://cassandra.link/[Cassandra.Link,window=blank]: Curated site with Cassandra articles, videos, and news feed.
```
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/pages/ecosystem.adoc]: with the addition of `cassandra.tools` it means that `cassandra.link` doesn't really belong in the "Cassandra Tools" section anymore. In fact I'm not sure where we would put `cassandra.link` on this page anymore?

File: site-content/source/modules/ROOT/pages/ecosystem.adoc [hunk 2]
```diff
@@ -346,6 +350,8 @@ https://opencredo.com/about-us/[Open Credo,window=blank]
 [openblock,accordion-content ml-medium hidden]
 --------
 
+https://youtube.com/playlist?list=PLmZzyjM-vqX6f0WQYhHgIv5K-esMRcbyr[Apache Cassandra Lunch,window=blank]
```
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/pages/ecosystem.adoc]: This is more content, than formal education (workshops, courses, etc). Like `cassandra.link` I am not sure where we should list this, open to ideas but the current list of categories doesn't really cater for either…?

**Review by michaelsembwever — CHANGES_REQUESTED** [review]
Two comments open regarding where the appropriate place for cassandra.link and the youtube playlist should be.

Other than that it looks good to me.

File: site-content/source/modules/ROOT/pages/ecosystem.adoc (hunk 2 above)
Comment (adp8ke) [reply site-content/source/modules/ROOT/pages/ecosystem.adoc]: @michaelsembwever Would you be open to creating a new category called "Cassandra Community Resources" or "Cassandra Community"? And we can put cassandra.link and Cassandra lunch into that category?

Comment (michaelsembwever) [reply site-content/source/modules/ROOT/pages/ecosystem.adoc]: @ErickRamirezAU what are your thoughts? what about "Community Blogs and Content" ?

Comment (ErickRamirezAU) [reply site-content/source/modules/ROOT/pages/ecosystem.adoc]: > @ErickRamirezAU what are your thoughts? what about "Community Blogs and Content" ?

I think the [Learn > Resources](https://cassandra.apache.org/_/resources.html) section of the website is a better fit for this. 🙂

**Comment by ErickRamirezAU** [comment]
Noting here that I've logged [CASSANDRA-17696](https://issues.apache.org/jira/browse/CASSANDRA-17696) for this PR. 🍻

File: site-content/source/modules/ROOT/pages/ecosystem.adoc (hunk 2 above)
Comment (michaelsembwever) [reply site-content/source/modules/ROOT/pages/ecosystem.adoc]: @ErickRamirezAU , are you thinking that Anant submits two tiles here (one for `cassandra.link` and one for the youtube playlist)? or that they submit each content and video piece they have as separate tiles on that page?

Comment (ErickRamirezAU) [reply site-content/source/modules/ROOT/pages/ecosystem.adoc]: +1 on doing 2 separate tiles. 👍

Comment (michaelsembwever) [reply site-content/source/modules/ROOT/pages/ecosystem.adoc]: @adp8ke are you good with that?

Comment (adp8ke) [reply site-content/source/modules/ROOT/pages/ecosystem.adoc]: @michaelsembwever Acknowledged, I am good with that. I will make the updates. Thank you!

**Comment by adp8ke** [comment]
@michaelsembwever I have moved Cassandra.Link and Cassandra Lunch to Learn > Resources

File: site-content/source/modules/ROOT/pages/resources.adoc [hunk 3]
```diff
@@ -798,4 +798,42 @@ https://www.youtube.com/watch?v=td5QRYOeGag&list=PLU2OcwpQkYCy_awEe5xwlxGTk5UieA
 ----
 //end card
 
+//start card  
+[openblock,card resource-card text-center shadow relative level-Beginner]
+----
+[openblock,title mt-medium]
+------
+[discrete]
+=== Cassandra.Link
```
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/pages/resources.adoc]: something more descriptive please (the reader needs a reason to click on it)

File: site-content/source/modules/ROOT/pages/resources.adoc [hunk 4]
```diff
@@ -798,4 +798,42 @@ https://www.youtube.com/watch?v=td5QRYOeGag&list=PLU2OcwpQkYCy_awEe5xwlxGTk5UieA
...
+----
+//end card
 
+//start card  
+[openblock,card resource-card text-center shadow relative level-Beginner]
+----
+[openblock,title mt-medium]
+------
+[discrete]
+=== Apache Cassandra Lunch Playlist
```
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/pages/resources.adoc]:
```suggestion
=== Apache Cassandra Lunch Youtube Playlist By Anant
```

File: site-content/source/modules/ROOT/pages/resources.adoc (hunk 3 above)
Comment (adp8ke) [reply site-content/source/modules/ROOT/pages/resources.adoc]: Acknowledged, I will make it more descriptive

**Comment by adp8ke** [comment]
@michaelsembwever following up on this in case it slipped through the cracks

**Review by michaelsembwever — APPROVED** [review]

**Comment by adp8ke** [comment]
@michaelsembwever thank you for approving. Who should I contact regarding merging this PR?

**Comment by adp8ke** [comment]
Hello @ErickRamirezAU, I tried to log into jira to comment on the ticket, but the reset password link is not working, perhaps due to the current ongoing jira issues. Can this PR be merged, or is anything else required?

**Comment by michaelsembwever** [comment]
@adp8ke the merge commit needs to be removed from the PR

(We don't merge in two directions. Only merge (fast-forward) PRs into trunk. To update your PR, rebase your work on top of the origin's trunk.)

**Comment by adp8ke** [comment]
@michaelsembwever Acknowledged, I will update this PR

**Comment by adp8ke** [comment]
@michaelsembwever going to close this PR and create another. Apologies.

**Pull request closed without merging**